saleor/tests/fixtures.py

```diff
@@ -1291,6 +1291,27 @@ def shipping_method_channel_PLN(shipping_zone, channel_PLN):
     return method
 
 
+@pytest.fixture
+def page_attribute(db):
+    attribute = Attribute.objects.create(
+        slug="page",
+        name="Page",
+        type=AttributeType.PAGE_TYPE,
+        input_type=AttributeInputType.SWATCH,
+        filterable_in_storefront=True,
+        filterable_in_dashboard=True,
+        available_in_grid=True,
+    )
+    return attribute
+
+
+@pytest.fixture
+def page_attribute_value(page_attribute):
+    return AttributeValue.objects.create(
+        attribute=page_attribute, name="Value", slug="value", value="test value"
+    )
```

Comment:
Do we need these new fixtures? We already have `page_type` examples of attributes: `size_page_attribute`, `tag_page_attribute, `author_page_attribute`